chainercv/transforms/bbox/crop_bbox.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+
+def crop_bbox(
+        bbox, y_slice=None, x_slice=None,
+        allow_outside_center=True, return_param=False):
+    """Crop bounding boxes.
```

Comment:
This is confusing because it sound as if bounding boxes are cropped into a smaller piece.

How about "Translate bounding boxes to a cropped region of an image."